src/theory/quantifiers/ce_guided_pbe.h

```diff
@@ -64,7 +87,42 @@ class CegConjecturePbe {
   void initialize( Node n, std::vector< Node >& candidates, std::vector< Node >& lemmas );
   bool getPbeExamples( Node v, std::vector< std::vector< Node > >& exs, 
                        std::vector< Node >& exos, std::vector< Node >& exts);
+  /** is PBE enabled for any enumerator? */
   bool isPbe() { return d_is_pbe; }
+  /** get candidate for enumerator */
+  Node getCandidateForEnumerator( Node e );
+  /** is the enumerator e associated with I/O example pairs? */
+  bool hasExamples( Node e );
+  /** get number of I/O example pairs for enumerator e */
+  unsigned getNumExamples( Node e );
+  /** get the input arguments for i^th I/O example for e, which is added to the vector ex */
+  void getExample( Node e, unsigned i, std::vector< Node >& ex );
+  /** get the output value of the i^th I/O example for enumerator e */
+  Node getExampleOut( Node e, unsigned i );
+  int getExampleId( Node n );
+  /** add the search val, returns an equivalent value (possibly the same) */
+  Node addSearchVal( TypeNode tn, Node e, Node bvr );
+  /** evaluate builtin */
+  Node evaluateBuiltin( TypeNode tn, Node bn, Node e, unsigned i );
+  
+private:
```

Comment:
Can you get rid of the mixed private/public sections?

Reply:
I'd prefer keeping the commits small. I can open a separate issue/PR to clean up this class?

Reply:
I've opened #1250.